content/06.quality_criteria.md

```diff
@@ -250,74 +322,14 @@ especially effective at the source code level.
 * **[QC.Sec04]** World-writable files or directories **MUST NOT** be present in the product’s
   configuration or logging locations.
 
-## 4.10. Code Workflow [QC.Wor]
+## 4.4. Code release
```

Comment:
If "Code Release" is eventually only composed of QC.Ver (see comment below about moving QC.Rev to "Code management"), it may be also be put under "Code management"

Reply:
agree and moved here